package.json

```
@@ -66,7 +66,7 @@
     "jsonwebtoken-esm": "^1.0.3",
     "media-chrome": "^0.16.0",
     "motion": "^10.14.3",
-    "rxjs": "^6",
+    "rxjs": "^7.0.0",
```

```suggestion
    "rxjs": "^7",
```
Use just the major so that renovate it don't try and bump the minor and patch ☺️